docs/experience/resources.md

```diff
@@ -0,0 +1,70 @@
+title: Resources Timing
```

Comment:
Resources -> Resource

Whenever you use the word **resource** with other words, it's not needed to have the plural of "resource". So, resource data and resource waterfall is correct, but also much easier to say compared to "resources data".